v3-docs/docs/cli/index.md

```diff
@@ -217,210 +217,123 @@ METEOR_CLIENT_ENTRYPOINT=client/main.js METEOR_SERVER_ENTRYPOINT=server/main.js
+**NPM packages:**
+- `@babel/runtime`, `meteor-node-stubs`, `vue`, `vue-meteor-tracker`, `vue-router`, `@types/meteor`, `@vitejs/plugin-vue`, `autoprefixer`, `meteor-vite`, `postcss`, `tailwindcss`, `vite`
 
-Meteor packages included: `meteor-base`, `mobile-experience`, `mongo`, `reactive-var`, `standard-minifier-css`,
+**Meteor packages:**
+- `meteor-base`, `mobile-experience`, `mongo`, `reactive-var`, `standard-minifier-css`,
 `standard-minifier-js`, `es5-shim`, `ecmascript`, `typescript`, `shell-server`, `hot-module-replacement`, `static-html`,
-`react-meteor-data`, `zodern:types`.
-
+`jorgenvatle:vite`
+</details>
 
-## --vue {vue}
+<details>
+<summary><strong>Minimal App</strong> (--minimal)</summary>
 
-The command `meteor create --vue app-name` creates a Meteor app with [Vue 3](https://vuejs.org/),
-[Tailwind CSS](https://tailwindcss.com), [Vite](https://vitejs.dev/), and [MongoDB](https://www.mongodb.com/). To
-create a complete app, including testing and deployment, follow the [Vue 3 tutorial](https://vue3-tutorial.meteor.com/).
+**NPM packages:**
+- `@babel/runtime`, `meteor-node-stubs`
 
-Npm packages included: `@babel/runtime`, `meteor-node-stubs`, `vue`, `vue-meteor-tracker`, `vue-router`, `@types/meteor`, `@vitejs/plugin-vue`, `autoprefixer`, `postcss`, `tailwindcss`, `vite`.
+**Meteor packages:**
+- `meteor`, `standard-minifier-css`, `standard-minifier-js`, `es5-shim`, `ecmascript`, `typescript`, `shell-server`,
+`static-html`, `webapp`, `ddp`, `server-render`, `hot-module-replacement`
+</details>
 
-Meteor packages included: `meteor-base`, `mobile-experience`, `mongo`, `reactive-var`, `standard-minifier-css`,
-`standard-minifier-js`, `es5-shim`, `ecmascript`, `typescript`, `shell-server`, `hot-module-replacement`, `static-html`,
-`vite:bundler`.
-
-You can also use Vue 3 with Vite by using the [jorgenvatle:meteor-vite](https://github.com/JorgenVatle/meteor-vite)
-package. You can see an example on the [meteor-vite](https://github.com/JorgenVatle/meteor-vite/tree/release/examples/vue)
-repository.
-
-Npm packages included: `@babel/runtime`, `meteor-node-stubs`, `vue`, `vue-meteor-tracker`.
-
-Meteor packages included: `meteor-base`, `mobile-experience`, `mongo`, `reactive-var`, `standard-minifier-css`,
-`standard-minifier-js`, `es5-shim`, `ecmascript`, `typescript`, `shell-server`, `tracker`, `static-html`, `akryum:vue-component`,
-`meteortesting:mocha`, `johanbrook:publication-collector`.
+::: tip File Structure
+To learn more about the recommended file structure for Meteor apps, check the [Meteor Guide](https://guide.meteor.com/structure.html#javascript-structure).
+:::
 
+::: tip Combining Options
+Most options can be combined. For example, you can create a TypeScript-based Vue app with the prototype packages using `meteor create --vue --typescript --prototype app-name`.
+:::
 
```

Comment:
I'm not sure this is true

Reply:
Good catch! I think the TypeScript option works only with React.